src/specs.ts

```diff
@@ -38,102 +31,77 @@ export async function specsFromOpts(opts: Opts): Promise<BenchmarkSpec[]> {
   for (const b of browsers) {
     if (validBrowsers.has(b) === false) {
       throw new Error(
-          `Browser ${b} is not yet supported, ` +
+          `Browser ${b} is not supported, ` +
           `only ${[...validBrowsers].join(', ')} are currently supported`);
     }
   }
 
-  const remoteUrls = [];
-  const localNames: {name: string, queryString: string}[] = [];
-  let anyLocalNamesAreStar = false;
```

Comment:
what's the plan to allow automatic running of a bunch of benchmarks?

Reply:
Use bash wildcards.